src/test/java/com/xiao/users/controller/UserControllerIntegrationTest.java

```diff
@@ -93,4 +98,64 @@ void testFindUserById_404() throws Exception {
                 .andExpect(jsonPath("$.errorMessage").value(String.format("User not found with the given input data id : '%s'", userId)));
     }
 
+    @Test
+    void testFindAllUser_200() throws Exception {
```

Comment:
Should add 1 one case with this scenario:

1. save 2 Users into database
2. perform findAllUser with page size is 1